app/main.go

```diff
@@ -19,34 +19,83 @@ import (
 	"errors"
 	"fmt"
 	"html/template"
+	"net"
 	"net/http"
-	"os"
 	"strconv"
 	"strings"
-	"time"
 
 	"github.com/google/ts-bridge/boltdb"
 	"github.com/google/ts-bridge/datastore"
+	"github.com/google/ts-bridge/env"
 	"github.com/google/ts-bridge/stackdriver"
 	"github.com/google/ts-bridge/storage"
 	"github.com/google/ts-bridge/tsbridge"
 
 	"github.com/dustin/go-humanize"
 	log "github.com/sirupsen/logrus"
+	"gopkg.in/alecthomas/kingpin.v2"
+)
+
+var (
+	debug = kingpin.Flag(
+		"debug", "enable debug mode",
+	).Envar("DEBUG").Default("false").Bool()
```

Comment:
Go style doesn't have a line length restriction, which avoids having to do this kind of line breaking making the code longer and (IMHO) less readable. Go style prefers to minimize height.

Reply:
I do agree it blurs a bit, how about I space those out?

I'm hesitant to make them one liners as `counterResetInterval` for example is 217 characters long and doesn't fit on my 31" screen, which makes it not very readable. Or maybe I'm missing something here?

Reply:
That might be a reasonable one to wrap individually. That doesn't mean all of these need to be wrapped.

Reply:
Gotcha. Unwrapped everything that's less than 120 lines and added spaces between large options for readability.